nncf/quantization/algo.py

```diff
@@ -1043,13 +1059,35 @@ def get_bit_stats(self):
         return table
 
     def _get_local_init_range_config(self, scope: Scope, scope_overrides: Dict[str, Dict],
-                                     global_init_range_config: Dict):
-        module_init_range_config = global_init_range_config
+                                     global_init_range_config: Dict, quantizer_group: str):
+        if isinstance(global_init_range_config, dict):
+           module_init_range_config = global_init_range_config     
+        else:
+            module_init_range_config = None
+            matched_init_range_config = []
+            for range_init_subconfig in global_init_range_config:
+                target_scopes = range_init_subconfig.get("target_scopes", None)
+                ignored_scopes = range_init_subconfig.get("ignored_scopes", None)
+                target_quantizer_group = range_init_subconfig.get("target_quantizer_group", quantizer_group)
+                if quantizer_group == target_quantizer_group and \
+                    (target_scopes is not None and in_scope_list(str(scope), target_scopes)  or\
+                    ignored_scopes is not None and not in_scope_list(str(scope), ignored_scopes)):
```

Comment:
In quantization builder, we have a "_should_consider_scope..." function that wraps this kind of logic. Do the same here - perhaps make a helper function that will be used both inside "_should_consider_scope..." and here, but taking different `ignored_scopes` and `target_scopes` arguments at runtime.

Reply:
Done.